### DISCUSSION TITLE
Security Onion Error when installing from USB

### DISCUSSION THREAD

**Discussion by anomaliesintent112** [Security-Onion-Solutions/securityonion#10025] [Unsupported Versions]
I just built a new computer for my home lab and im trying to install security onion on it but i keep getting a "Kernel Panic Not Syncing Fatal Error". I have attached a screenshot of my trace back unfortunately i am new at this and cant seem to find an error code to google that will give me any meaningful results. If you have encountered this before and know the solution or can just tell me what is breaking so i can go onto google and try to find an answer it would be much appreciated thank you!

Also sorry for the low quality my monitor isn't very good and it cuts off some of the text
![20230327_180553 - Copy](https://user-images.githubusercontent.com/109564945/228126319-7f7576a2-abcf-4cbb-ac7d-a575c044cce4.jpg)

**Comment by InfosecGoon** [comment]
It looks like your computer might have some hardware that's too new for the install kernel to handle. Are you able to install vanilla CentOS 7 on this machine?

**Reply by anomaliesintent112** [reply]
sorry for the delayed response life is crazy. I installed Ubuntu desktop and was able to use the network installer script to setup security onion. The only problem I have now is I don't know how to use it if you have any resources that you'd recommend id greatly appreciate it. Also i cant find squert which is weird to me

**Reply by InfosecGoon** [reply]
For resources, check out our documentation page and our YouTube channel, there's lots of good stuff there. "Security Onion Essentials" is a free training course available on YouTube.

https://docs.securityonion.net/en/2.3/index.html
https://www.youtube.com/@security-onion

Also, Squert was removed from the platform some time ago -- it was part of the old 16.04 version, but not 2.3.
